vite.config.js

```diff
@@ -40,7 +40,10 @@ export default defineConfig({
 	build: {
 		outDir: './build',
 		target: 'esnext',
-		commonjsOptions: { include: [] },
+		commonjsOptions: {
+			include: [],
+			esmExternals: true,
```

Comment:
Why did you need to add this extra field (`esmExternals`) to the vite configuration?

Reply:
Not sure about this but I have just removed it